### PR TITLE
Revised About Page

### PR DESCRIPTION
**Pull request by eharley19**

### CODE REVIEW

File: frontend/13f-app/src/components/pages/AboutPage.js [hunk 1]
```diff
@@ -1,32 +1,16 @@
...
-                <div className="profile">
-                <img src="https://via.placeholder.com/250"/>
-                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Libero, minima? Quod laboriosam, earum veritatis beatae ut nulla mollitia! Corporis debitis ea consequuntur sint recusandae minima omnis dolores, dignissimos illum non!</p>
-                <h4>github: <a>www.github.com</a></h4>
-                <h4>linkedin: <a>www.linkedin</a></h4>
+            <div className='container' id='about-main'>
+                <br></br>
+                <div className='bg-dark text-white col-md-8' id='story-panel'>
+                    <h3>About 13F Filing Searcher</h3>
+                    <p>This site was created through the collaboration of two aspiring software engineers, Rose Altianas (<a>www.github.com</a>) and Edward Harley (<a>www.github.com</a>), under the mentorship and supervision of two industry professionals.</p>
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/pages/AboutPage.js:11]:
```suggestion
                    <p>This site was created through the collaboration of two aspiring software engineers, <a href="https://github.com/rosealti">Rose Altianas</a> and <a href="https://github.com/eharley19">Edward Harley</a>, under the mentorship and supervision of two industry professionals.</p>
```

File: frontend/13f-app/src/components/pages/AboutPage.js [hunk 2]
```diff
@@ -1,32 +1,16 @@
...
-                <h4>github: <a>www.github.com</a></h4>
-                <h4>linkedin: <a>www.linkedin</a></h4>
-            </div>
-                <div className="profile">
-                <img src="https://via.placeholder.com/250"/>
-                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Libero, minima? Quod laboriosam, earum veritatis beatae ut nulla mollitia! Corporis debitis ea consequuntur sint recusandae minima omnis dolores, dignissimos illum non!</p>
-                <h4>github: <a>www.github.com</a></h4>
-                <h4>linkedin: <a>www.linkedin</a></h4>
+            <div className='container' id='about-main'>
+                <br></br>
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/pages/AboutPage.js:8]:
```suggestion
                <br />
```

**Comment by TheNorthEestern** [comment]
Check out `feature/revised-about-page-kacy` for some ideas.

File: frontend/13f-app/src/components/pages/AboutPage.js (hunk 1 above)
Comment (rosealti) [reply frontend/13f-app/src/components/pages/AboutPage.js:11]: Looks good. Do you need my github link?

File: frontend/13f-app/src/components/MainSection.css [hunk 3]
```diff
@@ -29,7 +29,36 @@
...
 }
 
-.date {
-    padding: 5px;
-    margin: 5px;
+@media screen and (max-width:480px) {
+    .main-container {
+        background: none;
+        background-color: rgba(132, 136, 138, 0.6);
+        height: 100vh;
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/MainSection.css:36]:
```suggestion
```

**Comment by TheNorthEestern** [comment]
You need to add the [`jsconfig.json`](https://github.com/secfilingsearcher/filing13F_searcher/blob/feature/revised-about-page-kacy/frontend/13f-app/jsconfig.json) file to the repository.

```json
{
  "compilerOptions": {
    "baseUrl": "src"
  },
  "include": ["src"]
}
```

File: frontend/13f-app/src/components/AboutPage.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    background-color: transparent;
+    border-bottom-color: white; 
+    border-bottom-style: none;
+    max-width: 200px;
+}
+
+div#story-panel > p {
+    font-family: 'Lato', sans-serif;
+    font-size: 1.10em;
+}
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/AboutPage.css:34]:
```suggestion
#about-main.container     {
    position: absolute;
    max-width: inherit;
    max-height: max-content;
    height: 100%;
    background-color: rgba(132, 136, 138, 0.6);
}

#story-panel {
    margin: 25px 25px 0px 25px;
    padding: 25px;
    border-radius: 25px;
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: space-around;
}

#story-panel a {
    font-weight: bold;
}

#story-panel .list-group li { 
    color: white;
    background-color: transparent;
    border-bottom-color: white; 
    border-bottom-style: none;
    max-width: 200px;
}

#story-panel > p {
    font-family: 'Lato', sans-serif;
    font-size: 1.10em;
}
```

File: frontend/13f-app/src/components/AboutPage.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-around;
+    }
+    div#story-panel > p {
+        font-size: .75em;
+    }
+    div#story-panel .list-group li { 
+        max-width: 150px;
+    }
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/AboutPage.css:66]:
```suggestion
    #about-main.container     {
        position: absolute;
        max-width: inherit;
        max-height: max-content;
        height: 100%;
        background-color: rgba(132, 136, 138, 0.6);
    }
    #story-panel {
        max-width:max-content;      
        margin: 10px 0px 10px 7px;
        padding: 10px;
        border-radius: 10px;
        display: flex;
        flex-direction: row;
        align-items: center;
        justify-content: space-around;
    }
    #story-panel > p {
        font-size: .75em;
    }
    #story-panel .list-group li { 
        max-width: 150px;
    }
```

File: frontend/13f-app/src/components/Navbar.css [hunk 6]
```diff
@@ -72,9 +72,9 @@
     display: flex;
     flex-direction: column;
     width: 100%;
-    height: 90vh;
+    height: 100vh;
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/Navbar.css:75]:
```suggestion
```

**Comment by TheNorthEestern** [comment]
@eharley19 You're still missing [this file](https://github.com/secfilingsearcher/filing13F_searcher/pull/179#issuecomment-977449154)

**Review by TheNorthEestern — APPROVED** [review]
The import path was incorrect, you actually _didn't_ need the `jsconfig.json` file.

**Pull request merged**